01-shapefile-attributes.Rmd

```diff
@@ -133,19 +133,22 @@ extent(x = point)
 
 ```
 
-#Shapefile Attributes
-
+#Querying Shapefile Attributes
 Shapefiles often contain an associated database or spreadsheet of values called
 `attributes` that describe the vector features in the shapefile. You can think
 of this like a spreadsheet with rows and columns. Each column in the spreadsheet
-is an individual `Attribute` that describes an object.
+is an individual `Attribute` that describes an object. Shapefile _attributes_ 
+include measurements that correspond to the geometry of the shapefile features.
 
-For example, the `roads_stream` shapefile contains an attribute called `TYPE`. Each
+For example, the `roads` shapefile contains an attribute called `TYPE`. Each
 line in the shapefile has an associated type which describes the type of road, or 
 element in the file. 
 
 
 You can quickly view all attributes of a spatial object in r using the `@data`. 
+Bonus: If you are comfortable using `slot()` it is another method to do the same
```

Comment:
Ok cool - why don't we open an issue on this one as well to just explore options and make a decision that we will apply to ALL of the lessons. in the raster lessons i've been trying to use methods as much as possible rather than slot calls. i'm less familiar with shapefiles in R however. Issue then we can accept this PR as is.